### ISSUE TITLE
Quick tutorial on how to use vmware-ova?

### ISSUE THREAD

**Issue by nitrocode** [spiegela/packer-builder-vmware-ova#1]
Hi. I'm using the vmware-iso builder currently to remotely upload and setup an ISO image on vmware. I have an OVA of the ISO already and it would be much easier to deploy the OVA and then run through the boot commands and provisions. 

I installed the "packer-builder-vmware-ova.exe" and put it in the %APPDATA%/packer.d/plugins directory. I set PACKER_LOG=DEBUG and when I validate the template, it says it discovers the vmware-ova plugin but then it doesn't recognize the "source_path" of the OVA file and it would seem it also does not recognize all of the remote configuration keys for a remote esx server.

Would you go into greater detail on how to get started using this new builder? I basically was hoping it would do everything the vmware-iso builder does except use an OVA instead of an ISO.

Thanks,

**Comment by spiegela** [comment]
Actually, I had forgotten that I had published this... I don't think I meant to.  I was attempting just to convert the OVA to an ISO, and piggy-back on the vmware-iso packer  builder.  It'd been a bit, but I recall that I had trouble modifying the configuration to chain the vmware-iso functions.  While I was troubleshooting my actual requirements changed, and I didn't need to use the OVA any longer...

Another option that would certainly work, would be to clone the vmware-iso builder, and just prepend the OVA conversion.  If you're interested in getting this working properly, I'm happy to help you out.

**Comment by nitrocode** [comment]
Yes! I'm very interested in getting this to work. I like your idea as well where you deploy the OVA, export the ISO, and then feed the ISO into the vmware-iso builder. However, it's going backwards a bit. 

The other option that you mentioned would be to clone the vmware-iso builder and instead of creating a VM with an ISO image attached, it would just deploy the OVA using the ovftool and then go through the boot commands, etc. 

This is completely new to me as well as the language, if you have some spare time, would you mind helping me with this?
